content/en/docs/reference/glossary/kubectl.md

```diff
@@ -7,12 +7,11 @@ short_description: >
   A command line tool for communicating with a Kubernetes cluster.
 
 aka:
-- kubectl
 tags:
 - tool
 - fundamental
 ---
-Command line tool for communicating with a Kubernetes cluster's
+Kubectl (or kubectl) is a command line tool for communicating with a Kubernetes cluster's
```

Comment:
1. Check that this change doesn't break any pages relying on the existing glossary text.
2. I prefer the original. Do we need to reword?